info/task_description.html

```diff
@@ -26,6 +26,6 @@
     <strong>Precondition:</strong>
     <li>number >= 0;</li>
     <li>number is integer;</li>
-    <li>B >= 0;</li>
+    <li>B > 0;</li>
```

Comment:
I think we should also remove 1 from the possible values of base B.
https://en.wikipedia.org/wiki/Radix

Reply:
in base 1
3 is 111
7 is 1111111
0 is
I am not sure however if base 1 is as legitimate as 2, 8, 10 etc.
If you decide to keep it, you need two tests - one I proposed (where any number is a palindrome) and one for zero (which is not a palindrome).
https://en.wikipedia.org/wiki/Unary_numeral_system